app/models/user.rb

```diff
@@ -28,6 +28,12 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :cohorts, class_name: 'Cohort', foreign_key: 'creator_id', dependent: :destroy
-  has_many :programs, class_name: 'Program', foreign_key: 'creator_id', dependent: :destroy
+  has_many :cohort_members, class_name: "CohortMember", foreign_key: "user_id", dependent: :destroy
+  has_many :mentors, class_name: "Match", foreign_key: "mentor_id", dependent: :destroy
```

Comment:
Can a user have many mentors? Will we allow a mentee to have one that one at a time?